src/deserialize/index.js

```diff
@@ -1,7 +1,7 @@
-import { flattenAttributes } from '../flattenAttributes'
 import { mapRelationships } from '../mapRelationships'
+import convertsToCamelCase from "../toCamelCase"
 
-export const deserialize = (resp) => {
+export const deserialize = (resp, options) => {
```

Comment:
Can we make this `import { setKeyTransform } from "../setKeyTransform"` and have `setKeyTransform` accept the deserialized object  and a `setKeyTransform` option?

Currently, we'll only support one key transform, the one you made, but let's call it `camelLower` as this matches the key transform of a lib that I've noticed people using this with: https://github.com/Netflix/fast_jsonapi